.github/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Let's remove license. We do not want to change project license without notifying users etc.
All aerogear projects have license files in the ROOT

Reply:
@wtrocki are not all with the LICENSE and it should be equals for all projects in this way was agreed to keep in this dir. Is easier for us if we centralized all in one place. However, please, feel free to remove it from the standards then we re-run the script.